action.yml

```diff
@@ -42,13 +42,18 @@ outputs:
 runs:
   using: "composite"
   steps:
+    - shell: bash
+      run: |
+        set -eo pipefail
+
+        download_url=$(curl https://api.github.com/repos/codacy/git-version/releases/latest | jq -r .assets[0].browser_download_url)
+        curl -Ls $download_url > bin/git-version
     - id: previous-version
       shell: bash
       run: |
         set -e
 
-        export PREVIOUS_VERSION=$(docker run -v $(pwd):/repo codacy/git-version:${{ inputs.tool-version }} \
-          /bin/git-version \
+        export PREVIOUS_VERSION=$(bin/git-version \
```

Comment:
you may need to run `git config --global --add safe.directory /repo` before running git version if you run into that error caused folder ownnership

Reply:
We're using Github Action git which should have all the permissions, no?

Reply:
But I can test it on pulse to see what happens.